iphone/Maps/LocalizedStrings/ar.lproj/Localizable.strings

```diff
@@ -265,7 +265,7 @@
 "instagram" = "إنستغرام";
 
 /* Text in the editor */
-"vk" = "VK";
+"vk" = "في كيه";
```

Comment:
@rtsisyk 
What are the correct next steps, if I disagree with this particular translation?